static/app/components/events/contexts/utils.tsx

```diff
@@ -249,6 +295,96 @@ export function getContextMeta(event: Event, type: string): Record<string, any>
   }
 }
 
+export function getFormattedContextData({
+  event,
+  contextType,
+  contextValue,
+  organization,
+  project,
+}: {
+  contextType: string;
+  contextValue: any;
+  event: Event;
+  organization?: Organization;
+  project?: Project;
+}) {
+  const meta = getContextMeta(event, contextType);
+
+  switch (contextType) {
+    case 'app':
+      return [
+        ...getKnownAppContextData({data: contextValue, event, meta}),
+        ...getUnknownAppContextData({data: contextValue, meta}),
+      ];
+    case 'device':
+      return [
+        ...getKnownDeviceContextData({data: contextValue, event, meta}),
+        ...getUnknownDeviceContextData({data: contextValue, meta}),
+      ];
+    case 'memory_info':
+    case 'Memory Info':
+      return [
+        ...getKnownMemoryInfoContextData({data: contextValue, event, meta}),
+        ...getUnknownMemoryInfoContextData({data: contextValue, meta}),
+      ];
+    case 'browser':
+      return [
+        ...getKnownBrowserContextData({data: contextValue, meta}),
+        ...getUnknownBrowserContextData({data: contextValue, meta}),
+      ];
+    case 'os':
+      return [
+        ...getKnownOperatingSystemContextData({data: contextValue, meta}),
+        ...getUnknownOperatingSystemContextData({data: contextValue, meta}),
+      ];
+    case 'unity':
+      return [
+        ...getKnownUnityContextData({data: contextValue, meta}),
+        ...getUnknownUnityContextData({data: contextValue, meta}),
+      ];
+    case 'runtime':
+      return [
+        ...getKnownRuntimeContextData({data: contextValue, meta}),
+        ...getUnknownRuntimeContextData({data: contextValue, meta}),
+      ];
+    case 'user':
+      return [
+        ...getKnownUserContextData({data: contextValue, meta}),
+        ...getUnknownUserContextData({data: contextValue, meta}),
+      ];
+    case 'gpu':
+      return [
+        ...getKnownGpuContextData({data: contextValue, meta}),
+        ...getUnknownGpuContextData({data: contextValue, meta}),
+      ];
+    case 'trace':
+      return [
+        ...getKnownTraceContextData({data: contextValue, event, meta, organization}),
+        ...getUnknownTraceContextData({data: contextValue, meta}),
+      ];
+    case 'threadpool_info':
+    case 'ThreadPool Info':
```

Comment:
how does this happen?

Reply:
`ThreadPool Info` and `Memory Info` are legacy versions of `threadpool_info` and `memory_info`. I added a comment to mention this.

We keep them around in case users haven't updated their SDKs to send the correct version of the context (but also this change was introduced like 3 years ago so IDK how relevant it still is, though I carried it over anyway)